nx-X11/programs/Xserver/hw/nxagent/x11.svg

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why are you adding those SVGs when they are not used by the code?

Reply:
> Why are you adding those SVGs when they are not used by the code?

As source reference. Bitmap icons are binary, SVGs are source files.